tests/console/kubeadm.pm

```diff
@@ -19,24 +19,20 @@ use utils "systemctl";
 sub run {
     select_console("root-console");
 
-    record_info 'etcd', 'Stop etcd and clean up';
-    systemctl 'disable --now etcd';
-    script_run 'rm -r /var/lib/etcd/*';
-
-    record_info 'Setup', 'Test: Package Installation';
-    my $packages = 'kubernetes-client kubernetes-kubelet kubernetes-kubeadm docker-kubic cri-tools';
-    trup_install($packages);
+    record_info 'Test #1', 'Test: Initialize kubeadm';
+    assert_script_run('kubeadm init --cri-socket=/var/run/crio/crio.sock --pod-network-cidr=10.244.0.0/16', 180);
```

Comment:
could you pass that to the `/dev/$serialdev`? Sometimes the output is too long and it doesn't fit at the screen of the needle as whole.

e.g. `| tee /dev/$serialdev";`

Also, feel free to add:

`assert_script_run "kubectl config view --flatten=true | tee /dev/$serialdev";`

in order to see the config that kubeadm does.